cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -79,6 +82,12 @@ def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=
 
     update_discussions_settings_from_course_task.delay(course_key_str)
 
+@receiver(COURSE_CATALOG_INFO_CHANGED)
+def listen_for_course_catalog_info_changed(sender, signal, **kwargs):
+    print(f"kwargs: {kwargs}")
+    send_to_event_bus(COURSE_CATALOG_INFO_CHANGED, f"course-catalog-info-changed", 'catalog_info.course_key',
```

Comment:
We should really have a different topic for dev, stage, and prod environments.

Reply:
I'm thinking that maybe the openedx-events interface can add an environment prefix/suffix.

Reply:
(From discussion elsewhere: It may actually happen in the implementation layer instead.)